scripts/tests/test_scripts.py

```diff
@@ -34,6 +36,7 @@ def test_embedding():
         '--epochs', '1', '--optimizer', 'sgd', '--ngram-buckets', '1000',
         '--max-vocab-size', '1000'
     ])
+    time.sleep(5)
```

Comment:
`subprocess.check_call` already waits for the process to complete. Why is `time.sleep` required here?

Reply:
The test_scripts might fail due to the multi-process testing. This might have a better chance to make the test run through.